### ISSUE TITLE
Bootloop on Moto G(6)

### ISSUE THREAD

**Issue by dhambel4** [phhusson/treble_experimentations#1129]
I am not able to boot AOSP 10.0 v212 on the Moto G(6). AOSP 9.0 works fine, however. I factory reset with TWRP then flashed the ARM A Only GAPPS package. It is stuck on the splash screen.

**Comment by phhusson** [comment]
Logs?
Use adb logcat

**Comment by dhambel4** [comment]
Interestingly enough, I can not get my device to run this command. When typing in "adb devices", it shows up as unauthorized, and I do not receive the RSA prompt asking me to allow access to this device. MTP and ADB debugging are both unabled, and there is no option to Revoke USB debugging.

**Comment by dhambel4** [comment]
both enabled*

**Comment by phhusson** [comment]
Do you have Magisk installed? it could lead to this behaviour

**Comment by dhambel4** [comment]
Yes, Magisk is installed. Should I remove it, then see what happens? or is there perhaps a module I can install?

**Comment by phhusson** [comment]
yes you should remove it and see what happens

**Comment by dhambel4** [comment]
Removal of magisk successful. My device is now authorized by ADB. I ran logcat, and it is spitting out a barrage of constant information. I suppose it is used for finding out hardware information real-time.

What exactly would you need from logcat?

**Comment by phhusson** [comment]
just send everything

**Comment by dhambel4** [comment]
So, interestingly enough, I can now boot into the OS just fine. I backed up both my Data and System partitions in TWRP, then wiped Data, Dalvik and Cache and flashed AOSP 10.0 and it booted into the home screen.

However, mobile data is not working. Neither is the home button or recent apps button; I can only go back.

I can make calls just fine, but when calling my phone, it goes straight to voicemail.

I can not enable developer options, as the build number is greyed out.

Since the original issue of bootlooping is fixed, I would be happy to start another thread if need be.

**Comment by dhambel4** [comment]
Second update - wiped Data, Dalvik and Cache and flashed the vanilla GSI. Everything now works. So, I am assuming that the version with superuser and google play services just does not work with the Moto G(6).
